EssentialFeed/EssentialFeedTests/Image Comments/API/ImageCommentsMapperTests.swift

```diff
@@ -0,0 +1,79 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+@testable import EssentialFeed
```

Comment:
Following from my previous comment, we won't need `@testable` by declaring a public initializer on `ImageComment`:
```suggestion
import EssentialFeed
```